### PR TITLE
Update the emergency fragment with the contact info

### PR DESCRIPTION
**Pull request by Suryansh1720001**
## Task #9 Completed Checklist

- [x] Formatted code for improved readability.
- [x] In the XML file, limited emergency contact to a maximum of 10 numbers with input type set to "number."
- [x] Enhanced user experience by removing unnecessary toasts. (A single toast efficiently conveys the message to the user.)

---

*Check the pull request for any questions or concerns, and proceed with merging if everything looks satisfactory.*
*Thank You!*

### CODE REVIEW

File: app/src/main/java/com/example/pehchaan/fragment/EmergencyFragment.java [hunk 1]
```diff
@@ -16,14 +12,17 @@
...
+
 import com.example.pehchaan.R;
 
 public class EmergencyFragment extends Fragment {
-    public int ec1 =0;
-    public int ec2 =0;
-    public int ec3 =0;
-    public int ec4 =0;
-    public int ecmsg =0;
+    public int ec1 = 0;
```
Comment (soumenkp2) [inline app/src/main/java/com/example/pehchaan/fragment/EmergencyFragment.java]: kindly rename these variables into meaningful names

File: app/src/main/java/com/example/pehchaan/fragment/EmergencyFragment.java [hunk 2]
```diff
@@ -61,7 +60,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         SharedPreferences.Editor editor1 = num1.edit();
 
         SharedPreferences num1g = this.getActivity().getSharedPreferences("phonenum1", Context.MODE_PRIVATE);
-        String value1 = num1g.getString("num1","Enter num1");
+        String value1 = num1g.getString("num1", "Enter Num1");
```
Comment (soumenkp2) [inline app/src/main/java/com/example/pehchaan/fragment/EmergencyFragment.java]: rename these value1, value2, value3 too.. into meaningful names for better understanding

File: app/src/main/java/com/example/pehchaan/fragment/EmergencyFragment.java [hunk 3]
```diff
@@ -224,41 +211,41 @@ else if(ec4>=2)
...
 
-                String number1 = num1.getString("num1","Enter Num1");
-                String number2 = num2.getString("num2","Enter Num2");
-                String number3 = num3.getString("num3","Enter Num3");
-                String number4 = num4.getString("num4","Enter Num4");
+                // A single toast efficiently conveys the message to the user.
+                Toast.makeText(getActivity(), "Message Sent",
+                        Toast.LENGTH_LONG).show();
+
+                String number1 = num1.getString("num1", "Enter Num1");
```
Comment (soumenkp2) [inline app/src/main/java/com/example/pehchaan/fragment/EmergencyFragment.java]: We can refactor these strings vars into a list of strings for better scalability in future.

File: app/src/main/java/com/example/pehchaan/fragment/EmergencyFragment.java (hunk 2 above)
Comment (Suryansh1720001) [reply app/src/main/java/com/example/pehchaan/fragment/EmergencyFragment.java]: okay

**Comment by Suryansh1720001** [comment]
Okay, after all changes, I'll update you soon.

**Comment by Suryansh1720001** [comment]
@soumenkp2 
Now review the changes and merge it accordingly.

**Pull request merged**